recipes/bert-extractive-summarizer/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "bert-extractive-summarizer" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bert-extractive-summarizer-{{ version }}.tar.gz
+  sha256: b316e0540b71df2c2120de2d5ccb97cdbd621b30fb0f4c4d93343a22b9365f3d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - transformers
+    - scikit-learn
+    - spacy
+
+test:
+  imports:
+    - summarizer
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/dmmiller612/bert-extractive-summarizer
+  summary: Extractive Text Summarization with BERT
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - 
```

Comment:
You need to add your github user name under the `recipe-maintainers`

Reply:
I added my GitHub user name under the recipe-maintainers section